.github/workflows/typespec-validation.yaml

```diff
@@ -0,0 +1,25 @@
+name: Typespec Validation
+
+on: pull_request
+
+jobs:
+  typespec-validation:
+    name: Typespec Validation
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
+
+    - name: Setup CI environment
```

Comment:
```suggestion
```

Reply:
The result does not show the title: 

![image](https://github.com/user-attachments/assets/754f179a-c38a-4cee-8f24-9b5fab98243f)

https://github.com/Azure/azure-rest-api-specs/actions/runs/11523690323/job/32082236524?pr=31033

Reply:
Updated my recommendation